docs/source/bricks.rst

```diff
@@ -38,7 +39,7 @@ You can find a full listing of optional kwargs in the documentation below.
 
 
   filename = os.path.join(EXAMPLE_DOCS_DIRECTORY, "layout-parser-paper-fast.pdf")
-  elements = partition(filename=filename)
+  elements = partition(filename=filename, content_type="application/pdf")
```

Comment:
the first example the user sees should probably be without `content_type="application/pdf"` as the easiest way to get started (assuming developer's eyes often just scan to the first code to copy and paste). maybe describe `content_type` in a comment below?